src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.html

```diff
@@ -28,7 +28,7 @@
          <div class="upload-button">
              <umb-button
                 type="button"
-                key="general_upload"
+                label-key="general_upload"

```

Comment:
Yep :) I didn't know either, until I tested it just to make sure... turns out this attribute is simply ignored if the other one is used